spec/cypress/integration/signup.spec.js

```diff
@@ -0,0 +1,103 @@
+import faker from 'faker'
+import { createSelector } from '../utils'
+
+const { name, internet, phone, company } = faker
+const firstName = name.firstName()
+const fullName = name.findName(firstName)
+const email = internet.email(firstName)
+// faker only allows for length and memorable options on its internet password object, so I can't guarantee there will be a number as well, so I'm hardcoding this
+const password = 'testpass1'
+// enforces XXX-XXX-XXXX format, which our front-end is enforcing in the application
+const phoneNumber = phone.phoneNumberFormat()
+const orgName = company.companyName()
+
+describe('Signup', () => {
+  beforeEach(() => {
+    cy.app('clean')
+    cy.server()
+    cy.route({
+      method: 'POST',
+      url: '/signup'
+    }).as('signup')
+    cy.visit('/signup')
+    cy.get(createSelector('organization')).type(orgName)
+    cy.get(createSelector('name')).type(fullName)
+    cy.get(createSelector('greetingName')).type(firstName)
+    cy.get(createSelector('multiBusiness')).click()
+    cy.get(createSelector('yesMultiBusiness')).click()
+    cy.get(createSelector('phoneType')).click()
+    cy.get(createSelector('homePhone')).click()
+    cy.get('[data-cy="languageEs"]').parent().click()
```

Comment:
A minor thing. Can we use `createSelector` here?

Reply:
Whoops, missed it 👍